tests/syscall/fs_linux/fs.edl

```diff
@@ -3,6 +3,8 @@
 
 enclave {
 
+    //from "sgxfs.edl" import *;
```

Comment:
Please remove unnecessary import. 

Also, can this file be removed entirely? It's exactly the same signature as provided by the parent and I'm trying to figure out if there's a way to reduce code duplication.

Reply:
Fixed, thanks!